packages/synapse-react-client/package.json

```diff
@@ -69,6 +69,7 @@
     "immutable": "4.1.0",
     "jotai": "^2.7.0",
     "json-rules-engine": "^4.1.0",
+    "jsonpath": "^1.1.1",
```

Comment:
We need to get all of the possible `properties` out of a JSON schema. The easiest way to do this is with JSONPath. The alternative is writing an algorithm to recursively walk through a JSON object.